src/app/services/bucket/qarnotBucketService.py

```diff
@@ -13,10 +14,23 @@ def connect_to_bucket(self):
         return self.bucket
 
     async def upload_file(self, bucket_key: str, file_binary: bin):
-        return await self.connect_to_bucket().add_file(file_binary, bucket_key)
+        try:
+            self.connect_to_bucket().add_file(file_binary, bucket_key)
+        except Exception as e:
+            print(e)
+            return False
+        return True
 
     async def get_uploaded_file(self, bucket_key: str):
-        return await self.connect_to_bucket().get_file(bucket_key)
+        try:
+            return self.connect_to_bucket().get_file(bucket_key)
```

Comment:
No need for await ?

Reply:
I got an error stating if I set it. as get_file returns a string a got a `object str can't be used in 'await' expression`